### ISSUE TITLE
React App Magenta Error × Unhandled Rejection (Error): slice() does not support negative begin indexing 

### ISSUE THREAD

**Issue by HarounAns** [tensorflow/magenta-js#352]
I'm having this issue when yarn installing the magenta libraries and trying to use them with my react app:

```
Unhandled Rejection (Error): slice() does not support negative begin indexing
```
![image](https://user-images.githubusercontent.com/20847319/64751682-ed4a0e00-d4ea-11e9-931a-459a004d5f6b.png)

The code is reproduced below:
```
import React from 'react';
import * as mv from '@magenta/music/node/music_vae';
import * as core from '@magenta/music/node/core';

export default class Magenta extends React.Component {
    constructor(props) {
        super(props);

        this.checkpoint = 'https://storage.googleapis.com/magentadata/js/checkpoints/music_vae/mel_16bar_small_q2';
        this.model = new mv.MusicVAE(this.checkpoint);
        this.player = new core.Player();

        this.state = { 
            date: new Date() 
        };
    }

    componentDidMount = () => {
        console.log("Got Model", this.model);
        console.log("Got player", this.player);

        this.model
            .initialize()
            .then(() => {
                // set vae temperature
                const vae_temperature = 1.5;

                // play with temperature passed in
                this.playVAE(vae_temperature);
            })
            
    }

    playVAE = (vae_temperature) => {
        if (this.player.isPlaying()) {
            this.player.stop();
            return;
        }

        this.model
            .sample(1, vae_temperature) // ERROR THROWN HERE
            .then((sample) => this.player.start(sample[0]));
    }

    render() {
        return (
            <div>
                Magenta
            </div>
        )
    }
}
```

**Comment by notwaldorf** [comment]
I think I merged a PR to fix this in master, but haven’t released a new release! Will do so tomorrow.

**Comment by HarounAns** [comment]
Did you release it yet? Its still running into the same hour

**Comment by notwaldorf** [comment]
I released 1.11.1 this week -- can you double check if this works for you now?

**Comment by notwaldorf** [comment]
Closing, this should be fixed with that release
